src/Control/Concurrent/Extra.hs

```diff
@@ -79,11 +79,11 @@ forkFinally action and_then =
 -- | Given an action, produce a wrapped action that runs at most once.
 --   If the function raises an exception, the same exception will be reraised each time.
 --
--- > let x ||| y = do t1 <- onceFork x; t2 <- onceFork y; t1; t2
--- > \(x :: IO Int) -> void (once x) == return ()
--- > \(x :: IO Int) -> join (once x) == x
--- > \(x :: IO Int) -> (do y <- once x; y; y) == x
--- > \(x :: IO Int) -> (do y <- once x; y ||| y) == x
+-- > let x ||| y = do t1 <- 'onceFork' x; t2 <- 'onceFork' y; t1; t2
```

Comment:
You can't write the identifier haddock markup under `>` blocks, only under `@` blocks, so I've undone these changes.